From 0d5756ee33e58713e3d4fce6a95134ca0f90d5a9 Mon Sep 17 00:00:00 2001
From: LuK1337 <priv.luk@gmail.com>
Date: Fri, 17 Nov 2017 12:28:43 +0100
Subject: [PATCH 02/25] msm8953-common: Replace QCOMs groups with oem_29XX

 * Currently, config.fs doesn't give the ability to add AIDs to Android
   init such that they can actually be used in the init scripts.
 * Replace all instances of qcom_diag, rfs and rfs_shared with the
   appropriate IDs (2901, 2903, 2904).

Change-Id: Ibc1a2db8c73ca6ff9d882e42ac734e4ea65ad949
---
 gps/etc/izat.conf | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/gps/etc/izat.conf b/gps/etc/izat.conf
index 2c923bf..2689e31 100644
--- a/gps/etc/izat.conf
+++ b/gps/etc/izat.conf
@@ -214,7 +214,7 @@ HARDWARE_TYPE=all
 PROCESS_NAME=lowi-server
 PROCESS_ARGUMENT=
 PROCESS_STATE=ENABLED
-PROCESS_GROUPS=gps net_admin wifi inet qcom_diag
+PROCESS_GROUPS=gps net_admin wifi inet oem_2901
 PREMIUM_FEATURE=0
 IZAT_FEATURE_MASK=0xf303
 PLATFORMS=all
@@ -236,7 +236,7 @@ HARDWARE_TYPE=all
 PROCESS_NAME=xtwifi-client
 PROCESS_ARGUMENT=
 PROCESS_STATE=ENABLED
-PROCESS_GROUPS=wifi inet gps rfs_shared system wakelock
+PROCESS_GROUPS=wifi inet gps oem_2904 system wakelock
 PREMIUM_FEATURE=1
 IZAT_FEATURE_MASK=0xf0f
 PLATFORMS=all
@@ -247,7 +247,7 @@ HARDWARE_TYPE=all
 PROCESS_NAME=slim_daemon
 PROCESS_ARGUMENT=
 PROCESS_STATE=ENABLED
-PROCESS_GROUPS=gps qcom_diag can
+PROCESS_GROUPS=gps oem_2901 can
 PREMIUM_FEATURE=1
 IZAT_FEATURE_MASK=0xf0
 PLATFORMS=all
-- 
2.7.4

